### ISSUE TITLE
Warning about ObjectSpace on Jruby

### ISSUE THREAD

**Issue by YurySolovyov** [rubyzip/rubyzip#274]
Happens on jruby 9.0.5.0

```
.../jruby/2.2.0/gems/rubyzip-1.1.7/lib/zip.rb:35: warning:
ObjectSpace impacts performance. 
See http://wiki.jruby.org/PerformanceTuning#dont-enable-objectspace
```

**Comment by creddy** [comment]
Hello @YuriSolovyov This has been fixed in https://github.com/rubyzip/rubyzip/pull/252 and a new version of the Gem has been released to rubygems.org

**Comment by YuriSolovyov** [comment]
Thanks for heads up, I'll try it and report back. Btw,  I think we need this in changelog.

**Comment by mnaberez** [comment]
@YuriSolovyov This was fixed by 8447f0e2302989615eb0141f31911fe36130f292 and released in [RubyZip 1.2.0](https://rubygems.org/gems/rubyzip/versions/1.2.0).  I submitted a pull request (#278) that adds the missing changelog entry.  This issue can be closed now.
